docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -3682,7 +3682,7 @@ Join an existing new Swarm
 
 - **200** – no error
 - **400** – bad parameter
-- **406** – node is already part of a Swarm
+- **406** – node is already part of a swarm
```

Comment:
could work, but I think I want to make that change in a follow up; I think we use it in quite some locations